GooglePayment/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,11 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.braintreepayments.api.googlepayment">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    package="com.braintreepayments.api.googlepayment">
+
+    <uses-sdk tools:overrideLibrary="com.braintreepayments.api,com.braintreepayments.api.core,com.paypal.android.sdk.onetouch.core,com.paypal.android.sdk" />
```

Comment:
This had to be added because the `minSdk` version of these SDKs is 21. We think this was always an issue, but we were using an older version of the Android gradle plugin that didn't pick it up.